ilmercatinodellibro/ilmercatinodellibro
# feat(client+server): fix login/registration UIs + handle #82 + smaller fixes

## What it does
This feature updates the UI of the Login and Registration pages

It also fixes: the following issues:
- #82
- Unneeded filters displayed in the Catalog page
- School course filters display order
- Other small spacing/color fixes

## How to test it
Go to the `/login` page
Click on "Register" or go to `/registration`
Login and check te drawer on the left
